sysutils/yabai/Portfile

```
@@ -57,12 +57,12 @@ destroot {
     }
 }
 
-set minxcodever 13.3
-if {([vercmp $xcodeversion ${minxcodever}] < 0)} {
+if {${os.major} < 20} {
     known_fail yes
     pre-fetch {
-        ui_error "${name} ${version} requires Xcode ${minxcodever} or greater to build."
-        return -code error "incompatible Xcode version"
+        # See https://github.com/koekeishiya/yabai/issues/1287
+        ui_error "${name} ${version} cannot build on macOS 11.0 and earlier."
```

Based on the condition, shouldn’t this instead be the following?

```suggestion
        ui_error "${name} ${version} cannot build on macOS 10.15 and earlier."
```